server/schedule/operator_controller_test.go

```diff
@@ -36,31 +36,31 @@ import (
 	"github.com/tikv/pd/server/schedule/operator"
 )
 
-func Test(t *testing.T) {
-	TestingT(t)
-}
-
-var _ = Suite(&testOperatorControllerSuite{})
+type operatorControllerTestSuite struct {
+	suite.Suite
 
-type testOperatorControllerSuite struct {
 	ctx    context.Context
 	cancel context.CancelFunc
 }
 
-func (t *testOperatorControllerSuite) SetUpSuite(c *C) {
-	t.ctx, t.cancel = context.WithCancel(context.Background())
-	c.Assert(failpoint.Enable("github.com/tikv/pd/server/schedule/unexpectedOperator", "return(true)"), IsNil)
+func TestOperatorControllerTestSuite(t *testing.T) {
+	suite.Run(t, new(operatorControllerTestSuite))
 }
 
-func (t *testOperatorControllerSuite) TearDownSuite(c *C) {
-	t.cancel()
+func (suite *operatorControllerTestSuite) SetupSuite() {
+	suite.ctx, suite.cancel = context.WithCancel(context.Background())
+	suite.NoError(failpoint.Enable("github.com/tikv/pd/server/schedule/unexpectedOperator", "return(true)"))
+}
+
+func (suite *operatorControllerTestSuite) TearDownSuite() {
+	suite.cancel()
 }
```

Comment:
Since this suite only creates a context inside it, I think we can remove it and change every test to use `testing.T`?